src/gatsby/on-render-body.js

```diff
@@ -46,32 +45,32 @@ export default ({ setHeadComponents }, pluginOptions) => {
     // TO DO - Warn about missing options
   }
 
-  let agent;
+  let fetchAgent;
```

Comment:
This needs to be outside the function definition to cache or this will reset its state on every call